src/api/NetworkErrorLink.js

```diff
@@ -0,0 +1,16 @@
+import { onError } from 'apollo-link-error';
+import logFormatter from '@/util/logFormatter';
+
+export default () => {
+	return onError(({
+		networkError,
+		operation,
+		forward
+	}) => {
+		if (networkError) {
+			logFormatter(operation, 'error');
+			logFormatter(networkError, 'error');
+		}
+		forward();
```

Comment:
Looking at examples, calling this doesn't appear necessary

Reply:
I thought so too! Especially looking at the code for the error link. However, when I initially tried without it it that made this a terminating link and halted the flow to other links...

Reply:
aha